app/javascript/components/Logout.jsx

```diff
@@ -2,12 +2,11 @@ import React, {useEffect} from "react";
 import { useNavigate } from "react-router-dom";
 import { getData, logout } from "./helpers/api_helpers";
 import { useError } from "./helpers/useError";
+import { useOutletContext } from "react-router-dom";
```

Comment:
Can this be ✂️? I don't see it being used anywhere.

Reply:
yep--next PR I'll do a pass on all components and make sure to remove any unused imports!